src/Layer/TiledGeometryLayer.js

```diff
@@ -276,6 +276,10 @@ class TiledGeometryLayer extends GeometryLayer {
 
         for (const e of context.elevationLayers) {
             const extents = node.getExtentsByProjection(e.projection);
+            const zoom = extents[0].zoom;
```

Comment:
Are all `extents` always the same zoom ?

Reply:
yes